src/xcode/ENA/ENAUITests/HealthCertificate/ENAUITests_13_CreateHealthCertificate.swift

```diff
@@ -573,32 +608,135 @@ class ENAUITests_13_CreateHealthCertificate: CWATestCase {
 
 		snapshot("screenshot_2g_plus_certificate_overview")
 	}
+	
+	func test_screenshot_ExportCertificatesTooltip() throws {
+		app.setLaunchArgument(LaunchArguments.infoScreen.healthCertificateInfoScreenShown, to: true)
+		app.setLaunchArgument(LaunchArguments.healthCertificate.recoveryCertificateRegistered, to: true)
+		app.setLaunchArgument(LaunchArguments.healthCertificate.shouldShowExportCertificatesTooltip, to: true)
+		app.launch()
 
-	func test_AdmissionStateChanges_Then_StateChangeIndicatorIsVisible() {
+		// Navigate to Certificates Tab.
+		app.buttons[AccessibilityIdentifiers.TabBar.certificates].waitAndTap()
+		
+		XCTAssertTrue(app.staticTexts[AccessibilityLabels.localized(AppStrings.Tooltip.ExportCertificates.title)].waitForExistence(timeout: .medium))
+		XCTAssertTrue(app.staticTexts[AccessibilityLabels.localized(AppStrings.Tooltip.ExportCertificates.description)].waitForExistence(timeout: .short))
+	}
+	
+	func test_screenshot_WhenMaskIsRequired_ForFederalStateBW() throws {
 		app.setLaunchArgument(LaunchArguments.healthCertificate.firstAndSecondHealthCertificate, to: true)
+		app.setLaunchArgument(LaunchArguments.healthCertificate.isDCCAdmissionCheckScenariosEnabled, to: true)
+		app.setLaunchArgument(LaunchArguments.infoScreen.healthCertificateInfoScreenShown, to: true)
+		app.launch()
+		
+		app.buttons[AccessibilityIdentifiers.TabBar.certificates].waitAndTap(.medium)
+		
+		// Check if the selection button exists
+		let selectionButtonCell = try XCTUnwrap(app.cells[AccessibilityIdentifiers.HealthCertificate.Overview.changeAdmissionScenarioCell])
+		
+		// Navigate to the scenario selection screen
+		selectionButtonCell.waitAndTap(.short)
+		
+		// Select federal state Baden Württemberg
+		app.cells.element(boundBy: 1).waitAndTap()
+		
+		snapshot("screenshot_mask_required_BW_certificate_overview")
+	}
+	
+	func test_screenshot_WhenMaskIsOptional_ForFederalStateBW() throws {
+		app.setLaunchArgument(LaunchArguments.healthCertificate.firstAndSecondHealthCertificate, to: true)
+		app.setLaunchArgument(LaunchArguments.healthCertificate.isDCCAdmissionCheckScenariosEnabled, to: true)
+		app.setLaunchArgument(LaunchArguments.infoScreen.healthCertificateInfoScreenShown, to: true)
+		app.launch()
+		
+		app.buttons[AccessibilityIdentifiers.TabBar.certificates].waitAndTap(.medium)
+		
+		// Check if the selection button exists
+		let selectionButtonCell = try XCTUnwrap(app.cells[AccessibilityIdentifiers.HealthCertificate.Overview.changeAdmissionScenarioCell])
+		
+		// Navigate to the scenario selection screen
+		selectionButtonCell.waitAndTap(.short)
+		
+		// Select federal state Baden Württemberg
+		app.cells.element(boundBy: 1).waitAndTap()
```

Comment:
Here we need to test what exactly should be displayed when mask is option, this is missing here.

Reply:
thanks, done.